scripts/load_meeting_prot.py

```diff
@@ -1,4 +1,7 @@
-"""Script to load info from trello and merge with info from latest meeting protocol."""
+"""This script loada info from trello and merge with info from latest meeting 
+protocol. It then loads a new meeting protocol with the merged info devided into
+comming and ongoing deliveries. The info consists of project name,project info, 
+flow cell ids and comments."""
 
 import bcbio.google
```

Comment:
Replace it by trello then :-)

I mean, time-wise, I am not sure at all that decoupling the Google part from scilifelab will be faster than moving the functionality. Also, even if it is slightly more difficult, at least is something that we will be willing to maintain